source/prereqs/prerequisites.rst

```diff
@@ -55,6 +55,15 @@ System and software
 
 .. note:: Python 3.4 has reached its end of life and is now deprecated. |pyATS| and the |library| no longer support Python 3.4.
 
+After you install Python, check to make sure that you have the following packages:

```

Comment:
For example, to install Python 3 in Ubuntu 16.04LTS, the follow packages are required to be installed:

Reply:
For example, or are these always needed for Ubuntu?

Reply:
hmm i think they might be required for ubuntu, but we don't own these, it would be basically "from our expertise"